tests/loaders/test_file_loader.py

```diff
@@ -8,56 +8,66 @@
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
-from os.path import abspath, join, dirname
+from os.path import abspath, dirname, join
 
-from unittest import TestCase
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)